dspace/config/modules/authentication-password.cfg

```diff
@@ -26,3 +26,22 @@
 # SHA-256, SHA-384, and SHA-512 should be available, but you may have 
 # installed others.  If not set, SHA-512 will be used.
 # authentication-password.digestAlgorithm = SHA-512
+
+###### Validate Password Robustness Configuration ######
+# (by default is enabled, to disable, either comment out this configuration or set it to an empty value)
+# This regular expression is used to validate password during creation of EPerson
+# or during the patch of password.
+# NOTE: when you configure a custom regex, you have to update the rules on angular's i18n end,
+# variable: "profile.security.form.notifications.error.robust-password".
```

Comment:
@Micheleboychuk : Small fix, but I think the "NOTE" here can now be removed.  The frontend is planned to be updated to just display the error message provided by the backend. So lines 34-35 here can be removed.